README.md

```diff
@@ -4,7 +4,7 @@
 </picture>
 
 # Realm Flipper Plugin
-This is a plugin for Flipper to debug Realm databases in react-native applications.
+This is a plugin for [Flipper](https://fbflipper.com/) to inspect Realm databases in [React Native](https://reactnative.dev/) apps.
```

Comment:
Can you only inspect and not update data?

Reply:
Good point. "inspect, modify and query" is not accurate